var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -44,11 +45,18 @@ class Vtk(CMakePackage):
     # At the moment, we cannot build with both osmesa and qt, but as of
     # VTK 8.1, that should change
     conflicts('+osmesa', when='+qt')
-    conflicts('^python@3:', when='@:8.0')
 
     extends('python', when='+python')
 
-    depends_on('python@2.7:', when='+python', type=('build', 'run'))
+    #Acceptable python versions depend on vtk version
```

Comment:
According to flake, all `#`-based comments need to start with a space like `# Acceptable python...`. Other than that this LGTM